Updated url and version number

jjasghar opened the pull request:
- Remove omnitruck because we have a redirect now
- We're running chef 13 now :metal:

Signed-off-by: JJ Asghar <jj@chef.io>

jamescott commented:
The description for the `node_name` setting is updated:

https://docs.chef.io/config_rb_knife.html#settings

Thanks for creating the PR. I merged it "by hand" in the linked commit above because it needed to also be in a couple other spots.

james

Commit:
Updated url and version number

- Remove omnitruck because we have a redirect now
- We're running chef 13 now :metal:

Signed-off-by: JJ Asghar <jj@chef.io>

Commit:
dtags replicate

Commit:
adds updated url to bootstrap documentation

brewn commented:
@jjasghar This link redirects me to the [Chef client download page](https://downloads.chef.io/chef) at the moment: https://chef.io/chef/install.ps1

cwebberOps commented:
@jjasghar I am 👎  on this. We want people using omnitruck.chef.io and NOT chef.io

Commit:
reverts URL changes while keeping updated version numbers
